### ISSUE TITLE
Inception V4 pre-trained model image labels mis-match

### ISSUE THREAD

**Issue by w19787** [tensorflow/models#2326]
i try to eval the pre-trained Inception v4 model which is provided in project readme. http://download.tensorflow.org/models/inception_v4_2016_09_09.tar.gz

with following the steps to created frozen pb and retrieve labels from ILSVRC-2012-CLS, i run following eval for the specific image: 
bazel-bin/tensorflow/examples/label_image/label_image \
  --image=~/ILSVRC2012_img_train_t3/n02085620/n02085620_7.jpeg \
  --input_layer=input \
  --output_layer=InceptionV4/Logits/Predictions \
  --graph=/tmp/frozen_inception_v4.pb \
  --labels=/tmp/imagenet_slim_labels.txt \
  --input_mean=0 \
  --input_std=255

the result is: 
(152): 0.937863:42.070668: I tensorflow/examples/label_image/main.cc:250] 153-'snow leopard, ounce, Panthera uncia'
 (159): 0.002344642.070691: I tensorflow/examples/label_image/main.cc:250] 160-'wire-haired fox terrier'
 (174): 0.001597582.070700: I tensorflow/examples/label_image/main.cc:250] 175-'marmoset'
 (274): 0.001103712.070708: I tensorflow/examples/label_image/main.cc:250] 275-'go-kart'
 (238): 0.001000292.070714: I tensorflow/examples/label_image/main.cc:250] 239-'canoe'

i have tested several images under n02085620, the result is similiar. the [153-'snow leopard, ounce, Panthera uncia'] is incorrect. it should be 173 -chihuahua.

can anyone tell me why this happens?

**Comment by michaelisard** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
